packages/graph/hash_graph/lib/graph/src/store/query/filter.rs

```diff
@@ -27,6 +31,47 @@ pub enum Filter<'q, T: QueryRecord> {
     ),
 }
 
+macro_rules! define_ontology_filters {
+    ($record:ty, $path:ty) => {
+        impl<'q> Filter<'q, $record> {
+            #[must_use]
+            pub const fn for_latest_version() -> Self {
```

Comment:
Do we want dosctrings for these, though?

Reply:
I added a docstring in https://github.com/hashintel/hash/pull/1257/commits/9c1414a1937a872f9ef7d8932c0a004c1b5a1e03
<img width="1018" alt="image" src="https://user-images.githubusercontent.com/21277928/198012301-d5933364-88f9-4b05-b5cd-8452a8bb47e7.png">

Reply:
With the new version we discussed off-line using a trait to get the path, the docstring for `for_latest_version` becomes "generic" for all instances. May be slightly confusing